### PR TITLE
API-10238 'list_agents' (v3.5) filter extended by 'suspended' field

### PR DESCRIPTION
**Pull request by lucas-sajdak**
# Links
- [Jira](https://livechatinc.atlassian.net/browse/API-10238)
- Preview: https://623086b58ee7343618841230--livechat-public-docs.netlify.app
# Description
Starting from configuration-api:v3.5 'list_agents' is able to filter using 'suspended' filter

### CODE REVIEW

File: src/pages/management/configuration-api/v3.5/index.mdx [hunk 1]
```diff
@@ -266,6 +266,7 @@ Returns all Agents within a license.
 | ------------------- | -------- | ---------- | ------------------------------------------------------------------------------------------------------------- |
 | `filters`           | No       | `object`   | Possible request filters                                                                                      |
 | `filters.group_ids` | No       | `int[]`    | IDs of the groups from which you want to list Agents. You'll only get the Agents from the specified group(s). |
+| `filters.suspended` | No       | `bool`     | If provided returns only Agents with the same value of field `suspended`.                                     |
```
Comment (oliwiapolec) [inline src/pages/management/configuration-api/v3.5/index.mdx:269]:
```suggestion
| `filters.suspended` | No       | `bool`     | If provided, returns only those Agents who have the same `suspended` field value.                                     |
```

**Comment by oliwiapolec** [comment]
Description is all good now, please also add the info about this new filter to the Changelog and we'll be all set :)

File: src/pages/management/changelog/index.mdx [hunk 2]
```diff
@@ -23,6 +23,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 ## [v3.5] - Developer preview
 
+### Agents
+
+- The [**List Agents**](/management/configuration-api/v3.5/#list-agents) method can now filter agents by `suspended` status.
```
Comment (oliwiapolec) [inline src/pages/management/changelog/index.mdx:28]:
```suggestion
- The [**List Agents**](/management/configuration-api/v3.5/#list-agents) method can now filter agents by their`suspended` status.
```

**Review by oliwiapolec — APPROVED** [review]

File: src/pages/management/changelog/index.mdx [hunk 3]
```diff
@@ -23,6 +23,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 ## [v3.5] - Developer preview
 
+### Agents
+
+- The [**List Agents**](/management/configuration-api/v3.5/#list-agents) method can now filter agents by their`suspended` status.
```
Comment (ale-kacperczyk) [inline src/pages/management/changelog/index.mdx:28]:
```suggestion
- The [**List Agents**](/management/configuration-api/v3.5/#list-agents) method can now filter agents by their `suspended` status.
```

**Pull request merged**